ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -85,10 +85,9 @@ if tf_support_is_available
 endif
 
 if tflite_support_is_available
-  if not get_option('flatbuf-support').disabled()
-    if not flatbuf_dep.found()
-      error ('flatbuf devel package should be install to build the tensorflow lite.')
-    endif
+  if not flatbuf_support_is_available
+    flatbuf_dep = dependency('flatbuffers', required : true, \
+        not_found_message : 'flatbuf devel package should be install to build the tensorflow lite subplugin.')
```

Comment:
Do you need to check flatbuffers here again?
LINE153 of /meson.build has already done it.

Reply:
I explained this issue to him face to face and he agreed with this code.